MapboxNavigation/NavigationViewController.swift

```diff
@@ -368,10 +369,11 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
             }
         }
         
-        if destination == nil {
-            let annotation = MGLPointAnnotation()
-            annotation.coordinate = route.coordinates!.last!
-            destination = annotation
+        for leg in route.legs {
+            if let last = leg.steps.last {
```

Comment:
This code only places an annotation at the last destination. Should we annotate the intermediate waypoints as well?